streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -1282,6 +1383,12 @@ trait ZStream[-R, +E, +A] extends Serializable { self =>
    */
   def runCollect: ZIO[R, E, List[A]] = run(Sink.collectAll[A])
 
+  /**
+   * Immediately ends the stream without consuming an element
+   */
+  def runDiscard: ZIO[R, E, Unit] =
+    self.discard.runDrain
```

Comment:
If we do this, all the stream does is execute its scopes and exit. Is it useful?

Reply:
Probaby not useful enough to warrant its own combinator. The usecase I had in mind is when we are handed a stream for somebody else and want to signal that we don't intend to process it any further